web/src/pages/dashboard.tsx

```diff
@@ -168,8 +169,21 @@ const DashboardPage = (props: Props): ReactElement => {
   };
 
   const renderQuickActions = (): ReactElement => {
```

Comment:
[sand] it doesn't have to be a part of this story, but as follow up, do you think this should be pulled into it's own component file, that way it can be tested in isolation we can reduce the amount of tests and code in the dashboard files.